yt_dlp/extractor/tenplay.py

```diff
@@ -1,70 +1,76 @@
 # coding: utf-8
 from __future__ import unicode_literals
-
+from datetime import datetime
+import base64
 from .common import InfoExtractor
 from ..utils import (
     HEADRequest,
     parse_age_limit,
-    parse_iso8601,
-    # smuggle_url,
+    urlencode_postdata,
 )
 
 
 class TenPlayIE(InfoExtractor):
     _VALID_URL = r'https?://(?:www\.)?10play\.com\.au/(?:[^/]+/)+(?P<id>tpv\d{6}[a-z]{5})'
+    _NETRC_MACHINE = '10play'
     _TESTS = [{
-        'url': 'https://10play.com.au/masterchef/episodes/season-1/masterchef-s1-ep-1/tpv190718kwzga',
+        'url': 'https://10play.com.au/todd-sampsons-body-hack/episodes/season-4/episode-7/tpv200921kvngh',
         'info_dict': {
-            'id': '6060533435001',
+            'id': 'tpv200928zskgi',
             'ext': 'mp4',
-            'title': 'MasterChef - S1 Ep. 1',
-            'description': 'md5:4fe7b78e28af8f2d900cd20d900ef95c',
-            'age_limit': 10,
-            'timestamp': 1240828200,
-            'upload_date': '20090427',
-            'uploader_id': '2199827728001',
+            'title': "Todd Sampson's Body Hack - S4 Ep. 2",
+            'description': 'md5:a73ea55671034c6367784405423ef5a0',
+            'age_limit': 15,
+            'timestamp': 1601379660,
+            'upload_date': '20200909',
+            'uploader_id': 'Channel 10',
         },
```

Comment:
Don't change test without good reason. Add as a new test case if needed. Also, the `id` of videos must be same as before so as not to break `--download-archive`

Reply:
looks like they no longer use brightcove. So it cant be helped